fishtest/fishtest/views.py

```diff
@@ -660,7 +660,8 @@ def purge_run(rundb, run):
 
     run['finished'] = False
     if 'sprt' in run['args'] and 'state' in run['args']['sprt']:
-      del run['args']['sprt']['state']
+      fishtest.stats.stat_util.update_SPRT(results,run['args']['sprt'])
+      run['args']['sprt']['state']=''
```

Comment:
@vdbergh You replace the `del` with assignment of an empty string. That is less code but also less clean up of objects?

Reply:
From my point of view the `'state'` field can have three values `''` (test isn't finished), `'accepted'` and `'rejected'`. 

So `run['args']['sprt']['state']='' ` seems somewhat more correct to me than deleting the state field and then elsewhere interpreting a missing state field as `''`.